app/views/promo_mailer/promo_activated_2018q1_verified.html.slim

```diff
@@ -32,8 +32,4 @@ hr style="border-top-width: 0px"
 
 p.promo--clear = t("promo.shared.tos_1")
 
-br= t("publisher_mailer.shared.signature_html")
```

Comment:
Can you take a look at the `switch_banner_1` here? It looks like it was removed so now it's causing this to encounter an error

Reply:
Adressed in this commit https://github.com/brave-intl/publishers/pull/1536/commits/c6db85bd227b5c29f9bbb3b45e4027d0c7635713